<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎游玩贪吃蛇</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>550</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>320</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-90</y>
       <width>501</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/gamepic/background.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>480</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>退出账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>380</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>排行榜</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>311</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/gamepic/snake_01.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>490</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>440</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏说明</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>430</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登录账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>500</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_8</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-11</x>
       <y>60</y>
       <width>651</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>480</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>350</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>550</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>积分</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>称号</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>0</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>480</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>20</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;蛇头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;蛇身&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>20</y>
       <width>31</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;墙&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>41</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>31</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;蛇&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>200</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>420</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>41</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>200</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>310</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>410</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>548</x>
       <y>514</y>
       <width>75</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>457</x>
       <y>20</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;食物&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>514</y>
       <width>75</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>地图设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>514</y>
       <width>75</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>画面设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>548</x>
       <y>514</y>
       <width>75</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>601</width>
       <height>491</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>201</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>默认地图（适合打榜）</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>25</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>天堑</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>265</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>随意几笔</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>265</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>笑脸</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>65</y>
        <width>240</width>
        <height>180</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>65</y>
        <width>240</width>
        <height>180</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>240</width>
        <height>180</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>240</width>
        <height>180</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
